bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/transactions/TransactionResourceManager.java

```diff
@@ -74,8 +74,10 @@
             TRANSACTION_PACKAGE_VERSION, "onRollback");
     private static final String ATOMIKOS_LOG_BASE_PROPERTY = "com.atomikos.icatch.log_base_dir";
     private static final String ATOMIKOS_LOG_NAME_PROPERTY = "com.atomikos.icatch.log_base_name";
+    private static final String ATOMIKOS_REGISTERED_PROPERTY = "com.atomikos.icatch.registered";
     private static final String CONFIG_TRANSACTION_MANAGER_ENABLED = "b7a.transaction.manager.enabled";
     private static final String CONFIG_TRANSACTION_LOG_BASE = "b7a.transaction.log.base";
+    private static final String USER_DIR = "user.dir";
```

Comment:
This property name should be available in runtime constants. Can you check. This is to avoid changes in multiple places

Reply:
Fixed in 3d0adf2

Reply:
You don't need this const isn't it

Reply:
yes. removed